src/v3/src/transformer/identify/transformIdentify.ts

```diff
@@ -21,9 +22,43 @@ import {
 import { getUsernameCookie } from '../../util';
 import { getUIElementWithName, removeUIElementWithName } from '../utils';
 
+const updateErrorMessages = (
+  dataSchema: Record<string, DataSchema>,
+  fieldName: string,
+  errorKey: string,
+) => {
+  // TODO: once OKTA-476303 is merged, remove this and
+  // add isCustomizedI18nKey check for adding keys here
+  const disabled = true;
+  if (disabled) {
+    return;
+  }
```

Comment:
I have to do this for now because there are integration tests that rely on a specific text, once the OK-PL/loc integration branch is merged I can apply the `isCustomizedI18nKey` function here to apply this override key.